docset/windows/hyper-v/Resize-VHD.md

```diff
@@ -55,21 +53,21 @@ If the VHD belongs to the virtual SCSI chain, you **can** resize the virtual dis
 PS C:\> Resize-VHD -Path c:\BaseVHD.vhd -SizeBytes 1099511627776
 ```
 
-Expands the VHDX to 1 terabyte (assuming that the previous size was less than the new size prior to the command).
+Expands the VHD to 1 terabyte if the previous size was less than 1 terabyte. If it was larger, the cmdlet will error because it cannot shrink a VHD.
```

Comment:
```suggestion
Expands the VHD to 1 terabyte if the previous size was less than 1 terabyte. If it was larger, the cmdlet will report an error because it cannot shrink a VHD.
```